ocw_data_parser/ocw_data_parser.py

```diff
@@ -287,11 +293,21 @@ def extract_foreign_media_locally(self):
             with open(path_to_containing_folder + file_name, "wb") as file:
                 response = get(media["link"])
                 file.write(response.content)
-            update_file_location(self.master_json, path_to_containing_folder + file_name)
+            if self.static_prefix:
+                update_file_location(self.master_json, self.static_prefix + file_name)
+            else:
+                update_file_location(self.master_json, path_to_containing_folder + file_name)
             log.info("Extracted %s", file_name)
         log.info("Done! extracted foreign media to %s", path_to_containing_folder)
         self.export_master_json()
 
+    def generate_static_site(self):
+        self.extract_media_locally()
+        self.extract_foreign_media_locally()
+        generate_html_for_course(
+            self.destination_dir + '/master.json',
+            self.destination_dir)
+
```

Comment:
can you add a unit test for this?  Usually repos are connected to codecov and travis, which help determine if new code has been tested and is working, but that hasn't been done yet for this repo.  Still a good idea to add unit tests for most changes though.

Reply:
For this maybe just mock `generate_html_for_course` and validate that calling `generate_static_site ` also calls the mock function with expected arguments.

Reply:
Can we hold off on this?  @pdpinch suggested adding codecov to this repo and I will do that, but I'd like to form a separate PR to form a set of unit tests that make sense for this project on the whole.  With the repo as it is, you can't even run the unit tests because of import pathing issues that are outside the scope of this PR.